### PR TITLE
Update checkout buttons and state

### LINKED ISSUE #176: "Checked out" is it a button or a status indicator?
image

"Checked out" is not a button but is styled exactly the same as the button next to it.

### PR DESCRIPTION
**Pull request by misolori**
Fixes #176

- Styled the "Checked Out" button to look more like a status indicator
- Disabled checkout buttons when clicked
- Added "Checking Out..." text while checkout procedure happens for a branch

gif

### CODE REVIEW

File: preview-src/index.ts [hunk 1]
```diff
@@ -131,14 +133,19 @@ function appendComment(comment: any) {
 
 function updateCheckoutButton(isCheckedOut: boolean) {
 	const checkoutButton = (<HTMLButtonElement>document.getElementById(ElementIds.Checkout));
+	const checkoutMasterButton = (<HTMLButtonElement>document.getElementById(ElementIds.CheckoutMaster));
 	checkoutButton.disabled = isCheckedOut;
-	checkoutButton.innerHTML = isCheckedOut ? `Checked Out` : `Checkout`;
+	checkoutMasterButton.disabled = false;
+	const activeIcon = '<svg class="octicon octicon-check" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>';
```
Comment (RMacfarlane) [inline preview-src/index.ts]: I think we need more handling for light themes, the icon and text color match the background on Light+

File: preview-src/index.css [hunk 2]
```diff
@@ -142,6 +142,16 @@ button:hover:enabled, button:focus:enabled {
 	cursor: pointer;
 }
 
+body button.checkedOut{
+	opacity: 1 !important;
+	background-color: transparent;
+}
+
+body button svg{
```
Comment (RMacfarlane) [inline preview-src/index.css]: nitpicky comment: could you add a space before the { here and above? 😄

File: preview-src/index.ts (hunk 1 above)
Comment (rebornix) [reply preview-src/index.ts]: We can probably add 

```css
.vs .octicon {
	color: white;
}
.vs-dark .octicon {
	color: black;
}
```

to the index.css

Comment (misolori) [reply preview-src/index.ts]: Yikes, I missed that. I'll use `var(--vscode-foreground)` for the text color instead. Thanks for catching that!

File: preview-src/index.css (hunk 2 above)
Comment (misolori) [reply preview-src/index.css]: @RMacfarlane you got it, updated the rest of the css to make sure there's a space before the `{`

**Pull request merged**